### PR TITLE
Update Basic.cpp

### PR DESCRIPTION
**Pull request by romanmia12**
Live a comment

### CODE REVIEW

File: Basic.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 int main ()
 {
    int a;
+   //Two and jobair
```
Comment (romanmia12) [inline Basic.cpp:5]: Leave a comment

**Comment by aljobair** [comment]
Confirm..

File: Basic.cpp (hunk 1 above)
Comment (aljobair) [inline Basic.cpp:5]: Ok

**Review by aljobair — COMMENTED** [review]
good

**Pull request merged**